task-definition.plugins.json

```diff
@@ -91,6 +91,10 @@
                 {
                     "name": "ASYNC_MIGRATIONS_DISABLE_AUTO_ROLLBACK",
                     "value": "1"
+                },
+                {
+                    "name": "KAFKA_PARTITIONS_CONSUMED_CONCURRENTLY",
+                    "value": "3"
```

Comment:
Can we deploy this separately from partitioning? This way when _something_ blows up we have an obvious revert target.

Reply:
yup, will split out - was thinking of doing this proactively anyway after the review